app/src/main/java/edu/usf/sas/pal/muser/util/EventUtils.java

```diff
@@ -0,0 +1,43 @@
+package edu.usf.sas.pal.muser.util;
+
+import android.content.Context;
+
+import com.simplecity.amp_library.ShuttleApplication;
+import com.simplecity.amp_library.model.Song;
+
+import edu.usf.sas.pal.muser.firebase.model.Event;
+import edu.usf.sas.pal.muser.firebase.model.SongData;
+
+
+
+public class EventUtils {
+
+    /**
+     * Function to populate the Event data class.
+     * @param song - The song for which the event occurred.
+     * @param capturedEvent - The event that was captured.
+     * @param context - The context of the Fragment.
+     * @return - Event object
+     */
+    public static Event newEvent(Song song, String capturedEvent, Context context){
+        long currentTimeMS = System.currentTimeMillis();
+        long nanoTime = System.nanoTime();
+        SongData songData = newSongData(song, context);
+        long elapsedTime =  System.currentTimeMillis() - song.startTime;
+        return new Event(capturedEvent, currentTimeMS, nanoTime, song.startTime, elapsedTime, songData);
+    }
+
+    /**
+     * Function to populate SongData class. Called by captureEvent().
+     * @param song - Details of the song.
+     * @param context - Context of the fragment.
+     * @return - SongData object
+     */
+    public static SongData newSongData(Song song, Context context){
```

Comment:
We don't need `context` here, right?

Reply:
Oh, nevermind, I just saw you other comment 👍

Reply:
Looks like we do because one of the arguments in line no, 39, and 41 requires the `context` to be passed as an argument. 

`song.getBitrateLabel(context)`